linalg/ode.hpp

```diff
@@ -530,27 +383,173 @@ class ESDIRK33Solver : public ODESolver
 class GeneralizedAlphaSolver : public ODESolver
 {
 protected:
-   mutable Vector xdot,k,y;
+   mutable Vector k,y;
```

Comment:
Can a comment maybe be made here to distinguished between this `Vector k` and the `StateData::std::vector<Vector> k`?

Reply:
Thanks for pointing out this potential point of confusion. I have renamed`StateData::std::vector<Vector> k`  to  `StateData::std::vector<Vector> data`. This is more descriptive and also solves the point raised.

This week I will also work on improved Doxygen commenting as suggested by @pazner.